tests/cephadm/test_cluster_view.py

```diff
@@ -0,0 +1,108 @@
+from json import loads
+
+from ceph.waiter import WaitUntil
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class ClusterViewError(Exception):
+    pass
+
+
+def verify_mon_service(host):
+    """
+    Verify MON service is up
+    Args:
+        host: Installer host
+    """
+    timeout, interval = 20, 6
+    for w in WaitUntil(timeout=timeout, interval=interval):
+        out = CephAdm(host).ceph.orch.ls(service_type="mon", format="json")
+        configs = loads(out)
+        for config in configs:
+            running = config.get("status").get("running")
+            size = config.get("status").get("size")
+        if running == size:
+            log.info(f"MON service: size: {size}, running: {running}")
+            return True
+        if w.expired:
+            raise ClusterViewError("MON service check Failed")
```

Comment:
Add meaning log: 

Mention why the check failed, or what check was intended and what went wrong.

Reply:
Updated with something like: f"MON service failed: expected: {size}, running: {running}"